app/models/term.rb

```diff
@@ -104,6 +107,16 @@ def check_fingerprint_count
     fingerprint.destroy if fingerprint&.terms&.count&.zero?
   end
 
+  # This is called before_destroy to avoid orphaning SuggestedResource records. Deleting terms should be an unlikely
+  # event, so this should come up rarely. If it does, it warrants the extra care to delete the record manually in the
+  # Rails console.
+  def check_suggested_resource
+    return unless suggested_resource
+
+    Sentry.capture_message('Cannot delete term with associated suggested resource')
+    throw :abort
```

Comment:
I'd suggest also adding something like `Rails.logger.error('Cannot delete term with associated suggested resource')` to allow more immediate feedback when run in a console (which seems like our likely current use).

development.rb seems to be missing a logging line that makes this actually work well since we upgrade to 7.2. But if I add [this back in](https://github.com/MITLibraries/tacos/pull/175/files#diff-d3c4b3f41072daa416f1920511e9b2e26caea8c5cec0a14cb9508589a4dafa47L77-L79) logging does seem to work.